test/e2e/mock-e2e.js

```diff
@@ -284,7 +284,7 @@ async function setupMocking(
     });
 
   await server
-    .forGet(`${GAS_API_BASE_URL}/networks/${chainId}/gasPrices`)
+    .forGet(new RegExp(`^${GAS_API_BASE_URL}/networks/\\d+/gasPrices$`, 'u'))
```

Comment:
no matter if we are on chain id 1337, now we are making a call to chain id 1 whenever we start a send/swap.


https://github.com/user-attachments/assets/25e0d7df-248d-4303-9f43-cda890a0559a

The gas API doesn't support chain 1337

![Screenshot from 2025-05-30 13-50-14](https://github.com/user-attachments/assets/7f088d54-6f6f-4a5a-b7ec-8aa668a9de49)